src/main/java/racingcar/service/CarService.java

```diff
@@ -0,0 +1,69 @@
+package racingcar.service;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Random;
+import racingcar.constants.CarConstant;
+import racingcar.constants.InputConstant;
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+
+public class CarService {
+    private Cars cars = new Cars(new ArrayList<Car>());
+    public final int RANDOM_NUM_MAX_VALUE = 10;
+
+    public void validateNameInput(String carNameInput) {
+        cars = new Cars(new ArrayList<Car>());
+        List<String> carNames = splitCarNames(carNameInput);
+        makeCar(carNames);
+    }
+
+    public void validateCountInput(int tryCount) {
+        if (tryCount <= CarConstant.TRY_COUNT_MIN.getNumber()) {
+            throw new IllegalArgumentException("[ERROR] 시도할 횟수는 0보다 큰 숫자여야 합니다.");
+        }
+    }
+
+    private void makeCar(List<String> carNames) {
+        for (String carName : carNames) {
+            Car car = new Car(carName, CarConstant.DISTANCE_DEFAULT.getNumber());
+            cars.addCarInformation(car);
+        }
+    }
+
+    public List<String> splitCarNames(String carNames) {
+        return List.of(carNames.split(InputConstant.CAR_NAME_SPLIT_DELIMITER.getMessage()));
+    }
+
+    public LinkedHashMap<String, Integer> runRound() {
+        LinkedHashMap<String, Integer> roundResult = new LinkedHashMap<String, Integer>();
```

Comment:
여기도 위 코멘트와 같이 변수의 타입을 변경해보면 어떨까요!

추가로 변수의 제네릭에 `타입 매개변수`를 이미 선언했다면 다음과 같이 초기화 부분에는 타입매개변수를 생략 가능합니다.

```suggestion
        LinkedHashMap<String, Integer> roundResult = new LinkedHashMap<>();
```